Ladybird/AppKit/UI/InspectorWindow.mm

```diff
@@ -0,0 +1,88 @@
+/*
+ * Copyright (c) 2023, Tim Flynn <trflynn89@serenityos.org>
+ * Copyright (c) 2024, Neil Viloria <neilcviloria@gmail.com>
+ *
+ * SPDX-License-Identifier: BSD-2-Clause
+ */
+
+#include <LibWeb/Cookie/Cookie.h>
+#include <LibWebView/Attribute.h>
+#include <LibWebView/InspectorClient.h>
+#include <LibWebView/ViewImplementation.h>
+
+#import <UI/Event.h>
+#import <UI/Inspector.h>
+#import <UI/InspectorWindow.h>
+#import <UI/LadybirdWebView.h>
+#import <UI/Tab.h>
+#import <Utilities/Conversions.h>
```

Comment:
Many of these includes/imports are unused

Reply:
I'll do some digging into my lsp configuration, it doesn't seem to show any warnings here but it probably should